### DISCUSSION TITLE
Termostato Differenziale con esphome

### DISCUSSION THREAD

**Discussion by dante70** [indomus/forum#212] [Hardware & Firmware]
buongiorno,
vorrei realizzare un termostato differenziale per comando pompa pannello solare.
Non ho approfondito tantissimo ma, non ho ai primi tentativi, trovato il modo di creare un automazione che come trigger abbia la differenza di 2 temperature.
Posto T1 e T2 le 2 temperature monitorate  io vorrei che si attivasse uno switch quando T1-T2>x dove x e' il mio set .
Voglio farlo con esphome in modo che non sia dipendente da homeassistant per maggior sicurezza se ho problemi di connessione wifi.
grazie

**Comment by Nik71git** [comment]
leggo solo ora, ma dovresti dare un'occhiata ai componenti climate di ESPHome. Probabilmente il bang-bang fa al caso tuo.
